.env

```diff
@@ -5,3 +5,4 @@ MYSQL_LOCAL_PORT=3308
 MYSQL_DB_DOCKER_PORT=3306
 SPRING_LOCAL_PORT=8080
 SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
.evn file shouldn't be added to git as usually contains confidential data, please remove from git and add to .gitignore